### PR TITLE
[Issue 360] Static Site Alert Component

### LINKED ISSUE #360: [Task]: Implement Alert for static site
### Summary

Now that the Front-end repo has been set up, we can use the template to start adding content for the future beta.grants.gov site. 
We ultimately want the design to match the [Figma mockup](https://www.figma.com/file/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov?type=design&node-id=0-1&mode=design&t=781AdnK0dT5hXDxT-0)

We are breaking up the work into components. This ticket focuses only on the Alert content. 
image

- Add the approved Alert content changes to the page
- Ensure all the links and form on the page works as expected
- Following the USWDS style guide

### Links
* [Figma mockup](https://www.figma.com/file/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov?type=design&node-id=0-1&mode=design&t=781AdnK0dT5hXDxT-0)
* [Static site content](https://docs.google.com/document/d/16l2Gy-AstT9cgJK3jy_qe0I53nHhgri1O3F9UVc1TTU/edit#heading=h.syecbpqvccr6)

### Acceptance criteria

- [x] Alert is working and match the Figma design
- [x] Appears as a story in Storybook
- [x] Test coverage
- [x] Static site content served from root of the application
- [x] Updated content and design to match the [Figma mockup](https://www.figma.com/file/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov?type=design&node-id=0-1&mode=design&t=781AdnK0dT5hXDxT-0) 
- [x] mobile formatting as expected
- [x] desktop formatting as expected
- [x] Accessibility testing (should be inherited when using USWDS)

### Dependencies
#283

### PR DESCRIPTION
**Pull request by SammySteiner**
## Summary
Fixes #360 

## Changes proposed

- Alert component 
- Alert has been added to index page
- Alert has it's own story
- Alert has it's own test and is tested on the index page

## Context for reviewers
- I'm assuming the Title section with "Home" will be replaced with the Hero component
- The mobile content for the alert in figma is different than the desktop content, I'm assuming that was an artifact of content changes, because the mobile content is longer. 🤷  But let me know if we want to accomodate different content for that piece based on screen size.

#### Desktop:
image

#### Mobile:
image

#### Docs
image

#### Single Story:
image

### CODE REVIEW

File: frontend/src/components/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Alert as USWDSAlert } from "@trussworks/react-uswds";
+
+type Props = {
+  type: "success" | "warning" | "error" | "info";
+  children: React.ReactNode;
+};
+
+const Alert = ({ type, children }: Props) => {
+  return (
+    <USWDSAlert type={type} headingLevel="h4" slim>
```
Comment (daphnegold) [inline frontend/src/components/Alert.tsx:10]: My first time working USWDS design library, and it's turning out to be awesome. Love we can just drop this in.

**Review by daphnegold — APPROVED** [review]
🚢 :shipit: 

This is so great! Awesome!@#$ 

The only thing I have to say that might warrant a teeny bit of consideration before merging is that the dropped in Alert component from USWDS has a colored line on the left side of the Alert. I personally think it looks fantastic, but it's not in the designs. So just a thought!

**Comment by SammySteiner** [comment]
> The only thing I have to say that might warrant a teeny bit of consideration before merging is that the dropped in Alert component from USWDS has a colored line on the left side of the Alert. I personally think it looks fantastic, but it's not in the designs. So just a thought!

Good catch! Based on your comment, I took a look at the truss USWDS React framework, as well as the USWDS design library, and the vertical line thing is hard coded into the design. Meaning, we'd have to write some custom Sass overrides to get rid of it. Based on that, I'm hoping it was just an oversight by Andy, and if he wants it removed, we can separate that off into its own ticket when he gets back.

**Review by acouch — CHANGES_REQUESTED** [review]
Looks great! This is my first time jumping in, similarly excited as @daphnegold .

Only issue I noticed is the icons not loading:

image

**Comment by acouch** [comment]
I tried to contribute a quick fix to the link on the index and broke the lint tests.

**Comment by SammySteiner** [comment]
> Looks great! This is my first time jumping in, similarly excited as @daphnegold .
> 
> Only issue I noticed is the icons not loading:
@acouch I'm not seeing that issue, and I tried running with docker using `make dev` as well as locally with `npm run dev`. Want to pair to see if we can figure out why it's not working on your machine?
@daphnegold did you see that issue with the logo not loading?

**Review by acouch — APPROVED** [review]

**Comment by acouch** [comment]
We discovered locally that there was another container of the same name, so the library wasn't downloaded.

**Pull request merged**